src/Devito.jl

```diff
@@ -7,23 +7,34 @@ const sympy = PyNULL()
 const devito = PyNULL()
 const devitopro = PyNULL()
 const seismic = PyNULL()
+const utils = PyNULL()
 
-has_devitopro() = haskey(PyCall.Conda._installed_packages_dict(), "devitopro")
+has_devitopro() = devitopro != devito
 
 function __init__()
     try
         copy!(numpy, pyimport("numpy"))
         copy!(sympy, pyimport("sympy"))
         copy!(devito, pyimport("devito"))
-        copy!(devitopro, (has_devitopro() ? pyimport("devitopro") : pyimport("devito")))
+        try
+            copy!(devitopro, pyimport("devitopro"))
+        catch e
+            copy!(devitopro, pyimport("devito"))
+        end
         copy!(seismic, pyimport("examples.seismic"))
     catch e
         if get(ENV, "JULIA_REGISTRYCI_AUTOMERGE", "false") == "true"
             @warn "unable to pyimport"
         else
             throw(e)
         end
-    end   
+    end
+    # Utilities. Need to both load and also add to PYTHONPATH
```

Comment:
This is "mandatory", in particular `subdom` will not be pickable (see slack for why it's needed) without it.